# file: C:\Users\Couch\AppData\Roaming\Python\Python313\site-packages\fastapi\routing.py
# hypothesis_version: 6.141.1

[400, '+json', '/', 'APIRoute', 'APIRouter', 'DELETE', 'GET', 'HEAD', 'JSON decode error', 'OPTIONS', 'PATCH', 'POST', 'PUT', 'Response_', 'Successful Response', 'TRACE', 'application', 'background', 'body', 'content-type', 'ctx', 'error', 'fastapi_astack', 'input', 'json', 'json_invalid', 'loc', 'model', 'msg', 'name', 'path', 'read_with_orm_mode', 'response', 'route', 'serialization', 'serialize', 'shutdown', 'startup', 'status_code', 'type', 'unknown']